### ISSUE TITLE
Trying to install FRASER 2.0 

### ISSUE THREAD

**Issue by mmartinezj** [gagneurlab/FRASER#81]
Hello, I'm trying to install FRASER 2.0, but when I install it with bioconductor, i get version 1.8.1, which is the first FRASER. When trying to use devtools i get the following error:

Using GitHub PAT from the git credential store.
Error: Failed to install 'unknown package' from GitHub:
  HTTP error 401.
  Bad credentials

  Rate limit remaining: 53/60
  Rate limit reset at: 2024-04-26 14:52:19 UTC

I have already checked my github path with Sys.getenv("GITHUB_PAT") and it seemes fine (I get [1] "" as answer). If I may have some help in order to install FRASER 2.0. 

Thank you in advance.

**Comment by ischeller** [comment]
Hi @mmartinezj , the latest FRASER 2.0 version is not yet on Bioconductor. We are in the process of pushing it there, so it should be available to install from Bioconductor soon. 
In the meantime, installation through devtools from github `(devtools::install_github('gagneurlab/FRASER')` seems to be working for me. Can you provide more details of your R / Bioconductor version and the command you used for installation?

**Comment by mmartinezj** [comment]
Thanks for your quick answer! I'm using R v4.2.0 and BiocManager v1.30.22. For installing FRASER with bioconductor i used:
if ("FRASER" %in% row.names(installed.packages()) == FALSE) BiocManager::install("FRASER")

For trying to install the latest version I tried both:
devtools::install_github('gagneurlab/FRASER', dependencies=TRUE)
and
devtools::install_github('gagneurlab/FRASER', ref='1.99.3', dependencies=TRUE)
both of them showing the same error from my last message.

If you need any more info, just let me know, thank you!

**Comment by ischeller** [comment]
As far as I can see, the error you get is not related to FRASER or the way you are trying to install it, but rather the setup of the GITHUB_PAT on your side.

Do you have a PAT from Github already? In that case, you would need to run `Sys.setenv(GITHUB_PAT = "your PAT")` (see e.g. [here](https://stackoverflow.com/questions/70908295/failed-to-install-unknown-package-from-github) or [here](https://github.com/r-lib/remotes/issues/330) for fixes for users with the same error).

**Comment by mmartinezj** [comment]
Hi, thanks for the help, as you said, it was a problem related with my GITHUB_PAT. I just created a new Token and afterwards I had no problems for downloading the package via devtools. The second link was pretty informative for helping me solve the issue. 

Also, in case anyone steps upon the same issue, I had to remove my previous instalation with BiocManager for the devtool command to work. Hope this helps!
